.travis.yml

```diff
@@ -1,5 +1,9 @@
 sudo: false
 language: python
 python: 2.7
-install: "pip install flake8"
-script: "flake8 ."
+env:
+  - TOXENV=tests
```

Comment:
This means we've switched from running flake8 to running the tests. Was this intentional? We should at least run flake8 in Travis and carefully consider if we want to run the tests (we do that in Jenkins, too).